Mozillians_Repositioning.md

```diff
@@ -0,0 +1,233 @@
+# Mozillians.org Repositioning Architecture
+
+* [Mozillians.org Services](#mozilliansorg-services)
+  + [Technologies (Common)](#technologies-common)
+  + [New Front-End](#new-front-end)
+  + [Mozillians.org](#mozilliansorg)
+  + [Search Service](#search-service)
+  + [Profile Publisher](#profile-publisher)
+  + [Orgchart Service](#orgchart-service)
+* [Integrity Checker](#integrity-checker)
+* [CIS requirements](#cis-requirements)
+  + [Profile Updates from Mozillians](#profile-updates-from-mozillians)
+  + [Profile Updates from external Sources](#profile-updates-from-external-sources)
+
+
+## Mozillians.org Services
+
+### Technologies (Common)
+
+#### GraphQL
+
+We stated a discussion around whether we want to test [GraphQL](https://graphql.org/) as our API
+query language. Could be a great fit and we plan to give it a shot.
+
+#### Node, Python, Rust
+
+We want to use whatever technology fits our needs. If it makes sense to write a small service in
+in Node or Rust we encourage it.
+
+
+### New Front-End
+
+A new front-end to render the new pages:
+
+- profile view
+- (edit profile section)
+  TBD if we do this _inline_ or as separate page
+- search result page
+- org chart
+
+#### Technologies
+
+- We plan to implement the new front-end using [Vue](https://vuejs.org).
+- We use components throughout across all pages
+- We plan to use global CSS instead of using components because we embrace the
+  [cascade](https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade)
+- Server side rendering is put on hold for now
+
+#### Questions
+
+Will we support IE11?
+
+#### Soft Goals
+
+- Encourage community engagement by having a simple lightweight dev setup
+  - Ideally we have mocks for all APIs
+- Have a non-JS fallback.
+  - Either server side rendering or
+  - Super basic html fallback
+
+
+### Mozillians.org
+
+Is still in charge of:
```

Comment:
how does the integration looks like between the two from a UX point of view?
links with redirects or separate URLs or something else?

note: reading further, it sounds like a reverse proxy would serve pages and redirect in the backend - but clarity on the technical aspect would be good